slash/react/src/Form/core/Field.tsx

```diff
@@ -146,15 +144,11 @@ export const Field = ({
     `af-form__${classNameSuffix}`,
     modifiers,
   );
-  const groupClassName = getComponentClassName(
-    className,
-    classModifier,
-    "af-form__group",
-  );
+  const groupClassName = getComponentClassName("af-form__group", classModifier);
 
   return (
     <div
-      className={classNames("row", groupClassName, {
+      className={classNames(className, "row", groupClassName, {
         "af-form__group--required": actualRequired,
         "af-form__group--label-top": labelPosition === "top",
       })}
```

Comment:
```suggestion
      className={classNames("row", groupClassName, {
"af-form__group--required": actualRequired,
"af-form__group--label-top": labelPosition === "top",
}, className)}
```
les class css de utilisateur doivent toujours être placé à la fin affin qu'elle soit en surcharge du css si besoin.